docs/input/docs/usage/examples.md

```diff
@@ -55,6 +55,25 @@ public class Data()
 }
 ```
 
+If a default value is needed, for example when specifying a file extension, the value can be encapsulated by a `Maybe` as well:
+
+```csharp
+/// Data class with mandatory property Foo and optional property MaybeBar. Default of MaybeBar is Maybe.None<Bar>.
```

Comment:
Something seems to be wrong here. There's neither a `MaybeBar` in the example nor a `Bar` type

Reply:
Thx, fixed it.